app/webpacker/controllers/toggle_controller.js

```diff
@@ -4,9 +4,22 @@ export default class extends Controller {
   static targets = ["content"];
 
   toggle(event) {
+    event.stopImmediatePropagation()
     const input = event.currentTarget;
+    const chevron = input.querySelector(".icon-chevron-down, .icon-chevron-up")
+    const toggleViaSingleElement = !input.dataset.toggleShow;
+
+    if(chevron) {
```

Comment:
I think if we made the chevron element an (optional) registered `target` of the controller we could simplify this a bit. So if we add `data-toggle-target: "chevron"` on the icon (and add `"chevron"` to the targets list), we could use the dynamically-defined ["existential" property](https://stimulus.hotwired.dev/reference/targets#properties) and just do:
```suggestion
    if(this.hasChevronTarget) {
```